tuxtype.appdata.xml

```diff
@@ -3,23 +3,20 @@
 <application>
   <id type="desktop">tuxtype.desktop</id>
   <name>Tux Typing</name>
+  <name xml:lang="pt_BR">Tecla Tux</name>
+  <name xml:lang="es">Teclea Tux</name>
   <summary>Educational typing tutor game starring Tux</summary>
   <summary xml:lang="de">Tastatur-Lernspielprogramm mit Tux</summary>
   <summary xml:lang="fr">Apprenez à taper au clavier avec Tux</summary>
   <summary xml:lang="ru">Kлавиатурный тренажер с пингвином в главной роли</summary>
   <summary xml:lang="tr">Daktilo Öğretmeni</summary>
+  <summary xml:lang="pt_BR">Ferramenta para crianças aprenderem a digitar</summary>
+  <summary xml:lang="es">Ahora niños se divertirán aprendiendo a teclear</summary>
   <project_license>GPL-3.0+</project_license>
   <metadata_license>CC0-1.0</metadata_license>
-  <description>
-    <p>
-      TuxTyping is an educational typing tutor for kids starring Tux, the Linux
-      penguin.
-    </p>
-    <p>
-      This educational game comes with two different games for practicing your
-      typing, and having a great time doing it.
-    </p>
-  </description>
+  <description><p>This fun application is a great way for kids, and even adults, to learn how to type! Learn the basics, or improve your skills by playing games that teach you how to use a keyboard without even looking at it. Children today are growing up surrounded by computers and mobile devices, and learning how to use these technologies well will only grow more important in the future. Start them off with a great foundation by downloading this app and encouraging them to play!</p></description>
```

Comment:
Where does this text come from?

Reply:
From the **Tux Typing** package in the **eos-apps** repository when it was available